crates/matrix-sdk-indexeddb/Cargo.toml

```diff
@@ -35,7 +35,8 @@ serde = "1.0.136"
 serde_json = "1.0.79"
 thiserror = "1.0.30"
 tracing = { workspace = true }
-wasm-bindgen = { version = "0.2.80", features = ["serde-serialize"] }
+wasm-bindgen = "0.2.83"
+gloo-utils = { version = "0.1", features = ["serde"] }
```

Comment:
Can we keep dependencies sorted alphabetically? 🙏🏼

Reply:
Sure!